tests/test_gc.rs

```diff
@@ -110,7 +111,7 @@ fn create_pointers_in_drop() {
     {
         let gil = Python::acquire_gil();
         let py = gil.python();
-        let empty = PyTuple::empty(py);
+        let empty: Py<PyTuple> = FromPy::from_py(PyTuple::empty(py), py);
         ptr = empty.as_ptr();
         cnt = empty.get_refcnt() - 1;
         let inst = Py::new(py, ClassWithDrop {}).unwrap();
```

Comment:
After calling `drop(inst)` the refcount of empty is increased by 4. Only 2 of those references survive the end of the scope.